module/helpers/aura.js

```diff
@@ -1,167 +1,76 @@
+        this.scene = scene;
+        // If auraData supplied is empty or null, fill it in with defaultAura
+        let auraData = mergeObject((scene.getFlag("arm5e", "aura") || {}), this.constructor.defaultAura, {overwrite: false});
+        mergeObject(this, auraData);
     }
-  ];
-  const activeEffectData = {
-    label: label,
-    icon: ICON,
-    duration: {
-      rounds: undefined
-    },
-    flags: {
-      arm5e: {
-        type: ["aura", "aura"],
-        subtype: ["auraLevel", "auraRealm"],
-        option: [null, null]
-      }
-    },
-    changes: changeData,
-    tint: "#000000"
-  };
-
-  return activeEffectData;
-}
-
-async function setAuraEffect(actor, activeEffectData) {
-  const aura = ArM5eActiveEffect.findAllActiveEffectsWithTypeFiltered(actor.effects, "aura");
-
-  if (aura.length > 1) {
-    console.error("Problem: multiple auras on an Actor");
-  }
-  if (aura.length) {
-    log(
-      false,
-      `AURA_MANAGEMENT Change aura for ${actor.name}, Aura ${activeEffectData.changes[0].value} ${activeEffectData.changes[1].value}`
-    );
-    aura[0]._getSourceName();
-    activeEffectData._id = aura[0]._id;
-    return await actor.updateEmbeddedDocuments("ActiveEffect", [activeEffectData]);
-  }
 
-  log(
-    false,
-    `AURA_MANAGEMENT Add aura for ${actor.name}, Aura ${activeEffectData.changes[0].value} ${activeEffectData.changes[1].value}`
-  );
-  return await actor.createEmbeddedDocuments("ActiveEffect", [activeEffectData]);
-}
-
-async function modifyAuraActiveEffectForAllTokensInScene(scene, value, type) {
-  if (!scene.active) {
-    return;
-  }
-  if (Object.values(ARM5E.REALM_TYPES).includes(type)) {
-    let activeEffectData = createAuraActiveEffect(type, value);
-    activeEffectData.origin = scene.uuid;
+    computeAuraModifierFor(alignment) {
+        let dominantRealm = this.dominantRealm
+        const multiplier = CONFIG.ARM5E.realmsExt[dominantRealm].influence[alignment];
+        this.computed = (this.values[this.dominantRealm] + this._nightModifier()) * multiplier;
+        return this.computed;
+    }
 
-    const tokens = scene.tokens.filter((token) => token.actor);
-    for (const token of tokens) {
-      if (token.actor._isCharacter()) {
-        if (token.isLinked) {
-          // patch the active effect data
-          await setAuraEffect(token.actor, activeEffectData);
-        } else {
-          // patch the active effect data
-          log(false, `Change aura for ${token.name}`);
-          await setAuraEffect(token.actor, activeEffectData);
+    get dominantRealm()
+    {
+        let dominantRealm;
+        let dominantRealmValue = 0;
+        for(let realm in this.values)
+        {
+            if (this.values[realm] > dominantRealmValue)
+            {
+                dominantRealmValue = this.values[realm];
+                dominantRealm = realm;
+            }
         }
-      }
+        return dominantRealm;
+        // TODO: What if two realms are the same value?
```

Comment:
What about this? 
![image](https://github.com/Xzotl42/arm5e/assets/28637157/16be9c3c-82be-4fe8-b879-3e718526a937)

Reply:
Iirc, a lacuna is a pocket aura in the middle of another aura. It is very circumstantial (eg: the place of a murder could have an infernal aura in an otherwise divine one).
V12 may help us implementing this